experiment/kind-e2e.sh

```diff
@@ -68,12 +66,19 @@ build() {
     fi
 
     # build the node image w/ kubernetes
-    kind build node --type=bazel
+    kind build node-image --type=bazel
 
     # make sure we have e2e requirements
     #make all WHAT="cmd/kubectl test/e2e/e2e.test vendor/github.com/onsi/ginkgo/ginkgo"
     bazel build //cmd/kubectl //test/e2e:e2e.test //vendor/github.com/onsi/ginkgo/ginkgo
 
+    # e2e.test does not show up in a path with platform in it and will not be found
+    # by kube::util::find-binary, so we will copy it to an acceptable location
+    # until this is fixed upstream
+    # https://github.com/kubernetes/kubernetes/issues/68306
```

Comment:
this should actually be fixed now

Reply:
ok deleting this comment + the mkdir + the cp